source/plugin/commands/arguments.rst

```diff
@@ -51,13 +51,13 @@ Example: Building a Command with Multiple Arguments
 
             .executor(new CommandExecutor() {
                 @Override
-                public CommandResult execute(CommandSource src, CommandContext args) throws CommandException {
-
+                public CommandResult execute(CommandSource src, CommandContext args)
+                                                                  throws CommandException {
```

Comment:
IMO you should use the Sponge code style that would intend this only 4 spaces after `public` (or 8? Can't remember right now)